### PR TITLE
Initialise project components

### PR DESCRIPTION
**Pull request by Zibilyonik**
Create a basic create react app with some working directory set up for the next milestone.
Set up linters.
Set Route and Switch for Bookstore component.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Zibilyonik 👋,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
-# redux-bookstore
\ No newline at end of file
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn start`
```
Comment (mikenath223) [inline README.md:7]: - [ ] Kindly update the readme and ensure it follows [this template](https://github.com/microverseinc/readme-template).

File: src/components/Bookstore.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const Bookstore = () => (
+  <Switch>
+    <Route exact path="/">
+      <div id="Books" />
+    </Route>
+    <Route path="/categories">
+      <h1>Under Construction</h1>
+    </Route>
+  </Switch>
```
Comment (mikenath223) [inline src/components/Bookstore.js:14]: - [ ] Kindly move these routes to the top level container `App.js`.

File: src/components/Bookstore.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Calculator.css';
+
+const Bookstore = () => (
+  <Switch>
+    <Route exact path="/">
+      <div id="Books" />
+    </Route>
+    <Route path="/categories">
+      <h1>Under Construction</h1>
```
Comment (mikenath223) [inline src/components/Bookstore.js:12]: - [ ] In App.js, the categories route should display the Categories component. So kindly create a Categories component and in it return the `Under construction` text.

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './App.css';
+
+function App() {
+  return (
```
Comment (mikenath223) [inline src/App.js:5]: - [ ] Please create a BookList component that renders a list of Books using the Book component.

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+    <div className="App" />
+  );
```
Comment (mikenath223) [inline src/App.js:7]: - [ ] Also, create a Book component that takes the book data as a prop and renders the title and the author.

Comment (mikenath223) [reply src/App.js:7]: - [ ] The Book component should also have a button for removing books, the button wouldn't remove the books yet, this is just a placeholder for now.

File: src/App.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+}
+
```
Comment (mikenath223) [inline src/App.js:9]: - [ ] Also add a CreateNewBook component it should render a form with inputs for the title and the author of the book, it should also have a button to submit, it won't actually submit but it's just a placeholder for now.

File: src/components/Bookstore.js (hunk 2 above)
Comment (mikenath223) [reply src/components/Bookstore.js:14]: - [ ] Also there should be two `<Link>`s in the navbar on the App.js container which leads to the booklist and categories locations. You can put the NavBar as a component.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**